assets/controllers/entries_navigation_controller.js

```diff
@@ -0,0 +1,58 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
```

Comment:
this got reworked a bit to be simpler.
also, before it was applied on all card list, list the imports page, where it has no sense. now it's applied only on the entries pages displayed as cards.
also, cross pages navigation was broken when using key shortcuts, now fixed